content/applications/finance/payment_acquirers/ogone.rst

```diff
@@ -5,84 +5,115 @@ Ogone
+configuration, as Odoo only allows a single **SHA-IN** and single **SHA-OUT** passphrase. In the
+**DirectLink and Batch** section of the screen, add the same **SHA-IN** passphrase as the previous
+field. You can leave the IP address field blank.
+
+When done, head to :menuselection:`Configuration --> Technical Information --> Transaction Feedback`
+and check the following options:
+
+- **I would like to receive transaction feedback parameters on the redirection URLs:** should be
+  checked
+- **Direct HTTP server-to-server request:** should to be set to *Online*
+- Both **URL** fields should contain the same following URL:
+  *https://<your-odoo-instance-url>/payment/ogone/accept*
+- **Request Method:** should be set to *Post*
+- **Dynamic eCommerce Parameters** should contain contain the following values: *ALIAS*, *AMOUNT*,
+  *CARDNO*, *CN*, *CURRENCY*, *IP*, *NCERROR*, *ORDERID*, *PAYID*, *PM*, *STATUS*, *TRXDATE*. Other
+  parameters can be included (if you have another integration with Ogone that requires them), but
+  are not advised.
+- The **URL** fields for **HTTP redirection in the browser** can be left empty, as Odoo will specify
+  these URLs for every transaction request.
+- In the **All transaction submission modes** section, fill in **SHA-OUT** passphrase and disable
+  **HTTP request for status change**.
+
+.. note::
+   - If you use tokenization (allowing users to save their cards), go to
+     :menuselection:`Configuration --> Alias` and set up the tokenization process according to your
+     needs.
 
 .. important::
-   If you are trying Ogone as a test, with the Test Account, change the **State** to *Test Mode*. We
-   recommend doing this on a test Odoo database, rather than on your main database.
+   - If you wish to run tests with Ogone, change the **State** of the Test Account to *Test Mode*.
+     We recommend doing this on an Odoo test database, rather than on your main database.
 
-Configuration on Ogone
-======================
+Settings in Odoo
+----------------
 
-Now that Odoo can communicate with Ogone, we need to make sure that Ogone can send information to
-your database.
+Credentials
+~~~~~~~~~~~
 
-To do so, log into your Ogone account and go to :menuselection:`Configuration --> Technical
-Information --> Transaction feedback --> Direct HTTP server-to-server request`.
+To set up Ogone in Odoo, head to :menuselection:`Accounting --> Configuration --> Payment Acquirers`
+or :menuselection:`Website --> Configuration --> Payment Acquirers` and open the Ogone acquirer. In
+the :guilabel:`Credentials` tab, enter the following information:
 
-Then, fill the form with the following data:
+- **PSPID:** The **PSPID** of your Ogone account
+- **API User ID:** Your **API user**'s ID (e.g. *OOAPI* if used conventionally)
+- **API User Password:** The password of the **API user**
+- **SHA Key IN/OUT:** The **SHA-IN** and **SHA-OUT** passphrases set in the backend of Ogone
```

Comment:
Is it necessary to explain the fields when the explanation is the same as the name of the field?

Maybe write at the line before
"enter the following information, as `configured on your Ogone portal <ref link to that paragraph above>`"